text-search/README.md

```diff
@@ -63,8 +63,7 @@ $ curl -s 'http://localhost:8080/search/?query=what+is+dad+bod'
 **Install python dependencies:**
 
 <pre data-test="exec">
-pip3 install -qqq --upgrade --force-reinstall pip
```

Comment:
We had to upgrade pip before. Otherwise there would be an error when installing tensorflow. But if that is not required anymore I think it is ok to remove it.

Reply:
you can still upgrade, but you should not use pip itself to upgrade, see https://github.com/pypa/pip/issues/5599 - feel free to add proper _python -m pip_ command or alternative